--- preps-1.7.0/configure.in.orig	Sun Jan 27 00:38:55 2002
+++ preps-1.7.0/configure.in	Sun Jan 27 00:39:57 2002
@@ -64,7 +64,7 @@
 [ ac_pgsql_dir=$withval ],
 ac_pgsql_dir=/usr/local/pgsql)
 
-pgsql_incdirs="$ac_pgsql_dir/include /usr/include/pgsql /usr/include /usr/local/include /opt/postgres/include"
+pgsql_incdirs="$ac_pgsql_dir/include /usr/include/pgsql /usr/include/postgresql /usr/include /usr/local/include /opt/postgres/include"
 AC_FIND_FILE(libpq-fe.h, $pgsql_incdirs, pgsql_incdir)
 if test ! -r $pgsql_incdir/libpq-fe.h; then
   AC_MSG_ERROR(libpq-fe.h not found.  May need to use --with-pgsqldir option)
