--- preps-1.9.6/configure.ac	Mon Aug  7 13:58:38 2000
+++ preps-1.9.6.new/configure.ac	Mon Nov  6 03:06:12 2000
@@ -71,15 +71,7 @@
    PG_CFLAGS="-I$pgsql_incdir"
 fi
 
-AC_MSG_CHECKING([PostgreSQL Libraries Installed])
-pgsql_libdirs="$ac_pgsql_dir/lib /usr/lib /usr/local/lib /usr/lib/postgresql/lib /opt/postgres/lib"
-AC_FIND_FILE(libpq.a, $pgsql_libdirs, pgsql_libdir)
-if test ! -r $pgsql_libdir/libpq.a; then
-  AC_MSG_RESULT(no)
-  AC_MSG_ERROR(libpq.a not found.  May need to use --with-pgsqldir option)
-fi
-AC_MSG_RESULT(yes)
-PG_LIBS="-L$pgsql_libdir -lpq"
+AC_CHECK_LIB(pq, main, [PG_LIBS="-lpq"])
 
 AC_SUBST(PG_CFLAGS)
 AC_SUBST(PG_LIBS)
