--- preps-1.8.0/doc/C/Makefile.am.orig	Sat Jul 21 05:43:28 2001
+++ preps-1.8.0/doc/C/Makefile.am	Wed Feb  6 07:45:35 2002
@@ -65,11 +65,12 @@
 
 ## install it all from the preps-manual docbook build dir.
 install-data-local: manuals.html
+	$(mkinstalldirs) $(DESTDIR)$(preps_helpdir)
 	-for file in $(srcdir)/manuals/*.html $(srcdir)/manuals/*.css; do \
 	  basefile=`basename $$file`; \
 	  echo $$file; \
 	  echo $$basefile; \
-	  $(INSTALL_DATA) $$file $(preps_helpdir)/$$basefile; \
+	  $(INSTALL_DATA) $$file $(DESTDIR)$(preps_helpdir)/$$basefile; \
 	done
 
 ## a couple other targets, just in case someone wants some other type of
y