diff -Naur preps-1.4.5/preps/mainwin.c preps-1.4.5-p/preps/mainwin.c
--- preps-1.4.5/preps/mainwin.c	Fri Sep  1 20:54:05 2000
+++ preps-1.4.5-p/preps/mainwin.c	Fri Oct 20 17:34:03 2000
@@ -129,13 +129,14 @@
  *    (COL_NUM + 1) : order by that column
  *   -(COL_NUM + 1) : order by that column in descending order
  */
-#define NUM_PR_FIELDS_DSP          6
+#define NUM_PR_FIELDS_DSP          7
 #define PR_NUM_DSP_POS             0
 #define PROJECT_NAME_DSP_POS       1
 #define STATUS_NAME_DSP_POS        2
 #define PROBLEM_TYPE_NAME_DSP_POS  3
 #define SEVERITY_NAME_DSP_POS      4
-#define PR_TITLE_DSP_POS           5
+#define PR_RESPONSIBLE_ID_DSP_POS  5
+#define PR_TITLE_DSP_POS           6
 
 /*
  *  The user can exit several ways.  This function centralizes the
@@ -357,6 +358,10 @@
             (strlen (PQgetvalue (res, i, PROJECT_NAME_POS)) + 1);
          strcpy (list_buf [PROJECT_NAME_DSP_POS],
                  PQgetvalue (res, i, PROJECT_NAME_POS));
+         list_buf [PR_RESPONSIBLE_ID_DSP_POS] = (gchar *)g_malloc
+            (strlen (PQgetbalue (res, i, PR_RESPONSIBLE_POS)) + 1);
+	 strcpy (list_buf [PR_RESPONSIBLE_ID_DSP_POS],
+                 PQgetvalue (res, i, PR_RESPONSIBLE_POS));
          list_buf [STATUS_NAME_DSP_POS] = (gchar *)g_malloc
             (strlen (PQgetvalue (res, i, STATUS_NAME_POS)) + 1);
          strcpy (list_buf [STATUS_NAME_DSP_POS],
@@ -1220,6 +1225,7 @@
   GtkWidget        *type_lbl;
   GtkWidget        *severity_lbl;
   GtkWidget        *title_lbl;
+  GtkWidget        *responsible_lbl;
   GtkWidget        *new_pr_button;
   GtkWidget        *mod_pr_button;
   GtkWidget        *find_pr_button;
@@ -1692,6 +1698,8 @@
   gtk_clist_set_column_width (GTK_CLIST (results_clist), 
                               SEVERITY_NAME_DSP_POS, 80);
   gtk_clist_set_column_width (GTK_CLIST (results_clist), 
+                              PR_RESPONSIBLE_ID_DSP_POS, 80);
+  gtk_clist_set_column_width (GTK_CLIST (results_clist), 
                               PR_TITLE_DSP_POS, 80);
   gtk_clist_column_titles_show (GTK_CLIST (results_clist));
   gtk_signal_connect (GTK_OBJECT (results_clist), "select_row",
@@ -1730,6 +1738,11 @@
   gtk_widget_show (title_lbl);
   gtk_clist_set_column_widget (GTK_CLIST (results_clist), 
                                PR_TITLE_DSP_POS, title_lbl);
+
+  responsible_lbl = gtk_label_new ("Responsible");
+  gtk_widget_show (responsible_lbl);
+  gtk_clist_set_column_widget (GTK_CLIST (results_clist), 
+                               PR_RESPONSIBLE_ID_DSP_POS, responsible_lbl);
 
   statusbar = gtk_statusbar_new ();
   gtk_object_set_data (GTK_OBJECT (main_win), STATUSBAR, statusbar);
diff -Naur preps-1.4.5/preps/pr_query.c preps-1.4.5-p/preps/pr_query.c
--- preps-1.4.5/preps/pr_query.c	Mon Aug  7 00:19:02 2000
+++ preps-1.4.5-p/preps/pr_query.c	Fri Oct 20 17:31:05 2000
@@ -32,7 +32,7 @@
 #define BASIC_SELECT "\
 SELECT prj.name, stat.name, pt.name, sevr.name, \
        pr.title, stat.order_num, sevr.order_num, \
-       pr.problem_num \
+       pr.problem_num, pr.responsible_id \
   FROM project prj, status stat, problem_type pt, \
        severity sevr, problem_report pr \
  WHERE prj.project_num = pr.project_num \
diff -Naur preps-1.4.5/preps/pr_query.h preps-1.4.5-p/preps/pr_query.h
--- preps-1.4.5/preps/pr_query.h	Sat Jun  3 15:07:39 2000
+++ preps-1.4.5-p/preps/pr_query.h	Fri Oct 20 17:31:05 2000
@@ -124,6 +124,7 @@
 #define STATUS_ORDER_POS        5
 #define SEVERITY_ORDER_POS      6
 #define PR_NUMBER_POS           7
+#define PR_RESPONSIBLE_POS      8
 
 __BEGIN_DECLS
 /*! \typedef pr_query_struct
