diff -Nur preps-1.4.5.orig/scripts/install_plpgsql.in preps-1.4.5/scripts/install_plpgsql.in
--- preps-1.4.5.orig/scripts/install_plpgsql.in	Fri Mar 17 05:07:24 2000
+++ preps-1.4.5/scripts/install_plpgsql.in	Sun Jan 21 15:26:51 2001
@@ -23,7 +23,7 @@
 -- Installing the PL/pgSQL language.
 ---------------------------------------------------------------------------
 CREATE FUNCTION plpgsql_call_handler () RETURNS OPAQUE AS
-        'XXPG_LIBXX/plpgsql.so' LANGUAGE 'C';
+        '/usr/lib/pgsql/modules/plpgsql.so' LANGUAGE 'C';
 
 CREATE TRUSTED PROCEDURAL LANGUAGE 'plpgsql'
         HANDLER plpgsql_call_handler
